.editorconfig

```diff
@@ -10,7 +10,7 @@ root = true
 indent_style = space
 ; (Please don't specify an indent_size here; that has too many unintended consequences.)
 charset = utf-8
-trim_trailing_whitespace = true
+trim_trailing_whitespace = false
```

Comment:
Please revert this.

Reply:
VS and VS Code reads this and automatically removes trailing whitespace. If I undo this change, the files that I'll edit after the revert will still have trailing whitespace removed.

Help me here...

Reply:
it should only do that if you select the lines. 
Additionally you can do all your changes with this setting and then before commiting (if you do it from the commandline), just revert this file back to the dev one. 

> git checkout dev -- .editorconfig

should do it.